### PR TITLE
Describe how to download pre-trained models

### PR DESCRIPTION
**Pull request by interfect**
The examples given in the README that you could copy-paste didn't work, because they didn't show how to download the pre-trained speech-to-text model. When you tried to run them you would get something like:

```
$ deepspeech output_model.pb ~/Desktop/testing.wav alphabet.txt
Loading model from file output_model.pb
Not found: output_model.pb; No such file or directory
Loaded model in 0.006s.
Running inference.
Segmentation fault
```

I've added some wget and tar commands to actually get the required trained models, so that a person starting with only pip and other standard Unix tools (plus a sound recorder) can actually get speech-to-text to happen.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,18 @@ Once installed you can then use the `deepspeech` binary to do speech-to-text on
 
 ```bash
 pip install deepspeech
-deepspeech output_model.pb my_audio_file.wav alphabet.txt
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz
```
Comment (lissyx) [inline README.md]: You could make that `wget -O https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz | tar xvfz - ` to simplify even more

Comment (lissyx) [reply README.md]: And you should not repeat the same command line over and over, it's going to get out of sync soon. Rather it would be better to add a section , say "Downloading inference material" and refer to that section of the README everywhere it might be needed :)

File: README.md [hunk 2]
```diff
@@ -113,9 +117,24 @@ $ pip install --upgrade deepspeech-gpu
 In both cases, it should take care of installing all the required dependencies. Once it is done, you should be able to call the sample binary using `deepspeech` on your command-line.
 
 ```bash
-deepspeech output_model.pb my_audio_file.wav alphabet.txt lm.binary trie
+deepspeech -h
```
Comment (lissyx) [inline README.md]: Why this change ?

File: README.md [hunk 3]
```diff
@@ -113,9 +117,24 @@ $ pip install --upgrade deepspeech-gpu
...
 ```
 
+To actually use the binary without doing your own training, you will need to download the pre-trained models:
+
+```bash
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz
+tar -xvzf deepspeech-0.1.0-models.tar.gz
+```
+
+Then, for an input 16-bit 16 kHz WAV file `my_audio_file.wav`, you can run inferrence:
```
Comment (lissyx) [inline README.md]: This is not completely accurate and already documented somewhere else. We should refer to that instead of copy/pasting the same information.

**Comment by mikehenrty** [comment]
This is a great start @interfect, and thanks for noticing this issue! I hope you don't mind, I'm going to cherry pick your commit, and do the things @lissyx asked so we can get your fix in.

**Comment by interfect** [comment]
Thanks! That's fine with me.
